CHANGELOG.md

```diff
@@ -1,15 +0,0 @@
-# 0.1.0
```

Comment:
I understand that this changelog has a different format than the changelog in our other trackers but this is the format that is used in Dart packages and I think it is necessary for pub.dev to create a nice [Changelog section](https://pub.dev/packages/snowplow_tracker/changelog).

Question: Why did you remove the alpha releases from the changelog? They were also published on pub.dev so I think it makes sense to have them here.

Reply:
Oh ok! I removed them as part of standardising with other changelogs. Will revert.